install/0000_80_machine-config-operator_01_machineconfigpool.crd.yaml

```diff
@@ -227,7 +227,6 @@ spec:
                     type: object
                     required:
                       - bundle
-                      - expiry
```

Comment:
non-blocking: was this removed to match somewhere else?

(also looks like the description right above has a typo)

Reply:
this was John who moved this, there's a bug with making everything required. The CRDs are vendored in from API so I can't change this